openlibrary/core/vendors.py

```diff
@@ -43,92 +44,83 @@ def search_amazon(title='', author=''):
 
     results = lending.amazon_api.search(Title=title, Author=author, SearchIndex='Books')
     data = {'results': []}
-    try:
-        for product in results:
-            data['results'].append(_serialize_amazon_product(product))
-    except SearchException:
-        data = {'error': 'no results'}
+    #try:
+    #    for product in results:
+    #        data['results'].append(_serialize_amazon_product(product))
+    #except SearchException:
+    #    data = {'error': 'no results'}
     return data
 
-
 def _serialize_amazon_product(product):
     """Takes a full Amazon product Advertising API returned AmazonProduct
     with multiple ResponseGroups, and extracts the data we are interested in.
 
     :param amazon.api.AmazonProduct product:
     :return: Amazon metadata for one product
     :rtype: dict
-    """
-
-    price_fmt = price = qlt = None
-    used = product._safe_get_element_text('OfferSummary.LowestUsedPrice.Amount')
-    new = product._safe_get_element_text('OfferSummary.LowestNewPrice.Amount')
 
-    # prioritize lower prices and newer, all things being equal
-    if used and new:
-        price, qlt = (used, 'used') if int(used) < int(new) else (new, 'new')
-    # accept whichever is available
-    elif used or new:
-        price, qlt = (used, 'used') if used else (new, 'new')
-
-    if price:
-        price = '{:00,.2f}'.format(int(price)/100.)
-        if qlt:
-            price_fmt = "$%s (%s)" % (price, qlt)
-
-    data = {
+    {
+      'price': '$54.06',
+      'price_amt': 5406,
+      'binding': 'Hardcover',
+      'authors': {'Greenfield, Ben': 'Author'},
+      'publication_date': '2020-01-21T00:00:01Z',
+      'dimensions': {
```

Comment:
For dimensions OL uses 
```
"physical_dimensions": {
      "type": "string",
      "examples": [
        "5.4 x 4.7 x 0.2 inches",
        "21 x 14,8 x 0,8 centimeters"
      ]
}
```

and

```
"weight": {
      "type": "string",
      "examples": [
        "300 grams",
        "0.3 kilos",
        "12 ounces",
        "1 pounds"
      ]
}
```

(from https://github.com/internetarchive/openlibrary-client/blob/master/olclient/schemata/edition.schema.json )

Reply:
Dropping dimensions for now.